modules/cdc-ext/src/main/java/org/apache/ignite/cdc/kafka/IgniteToKafkaCdcStreamer.java

```diff
@@ -213,6 +213,16 @@ public IgniteToKafkaCdcStreamer(
 
     /** {@inheritDoc} */
     @Override public void start(MetricRegistry mreg) {
+        if (kafkaProps == null)
+            throw new IllegalStateException("The Kafka properties must be set in order to initiate a connection.");
+
+        kafkaProps.setProperty(KEY_SERIALIZER_CLASS_CONFIG, IntegerSerializer.class.getName());
+        kafkaProps.setProperty(VALUE_SERIALIZER_CLASS_CONFIG, ByteArraySerializer.class.getName());
+
+        cachesIds = cacheNames.stream()
```

Comment:
We should check `A.notEmpty(cacheNames)` on start.

Reply:
It seems that we can check it right in the setter. Done.